tests/ui/inference/erase-type-params-in-label.stderr

```diff
@@ -2,7 +2,7 @@ error[E0283]: type annotations needed for `Foo<i32, &str, W, Z>`
   --> $DIR/erase-type-params-in-label.rs:2:9
    |
 LL |     let foo = foo(1, "");
-   |         ^^^   --- type must be known at this point
+   |         ^^^   ---------- type must be known at this point
```

Comment:
This is an orthogonal issue but seems worth mentioning: it took me a while to realize that these `---` sequences are pointing to the code above them. It's much less obvious than the `^^^` sequences. I just interpreted it as text to lead the reader's eye towards the `^^^`.

Reply:
It's likely I've been overlooking the spans demarcated by these `---` markers for  years.

Reply:
😮 

I'm really surprised to hear! They mark the secondary spans, usually giving you extra context. *At least*, even if you didn't realize that they have the benefit of bringing interesting code into view (if it didn't happen to be in the lines of the primary span).

Reply:
I'm sure people have suggested using non-ASCII chars for this sort of thing, and presumably there's a reason why it's still all ASCII, e.g. lack of reliable support in all terminals.

Reply:
It's kind of blocked on having a `rustc.toml` file where we could customize output, so that people can change their configuration without much hassle when in less supported platforms. We should do this sooner rather than later. :-/